src/app/pages/common/entity/entity-form/components/form-chip/form-chip.component.ts

```diff
@@ -40,14 +41,15 @@ export class FormChipComponent implements Field, OnInit {
       if (this.chipLists !== this.group.controls[this.config.name].value && typeof this.group.controls[this.config.name].value === 'object') {
         this.chipLists = this.group.controls[this.config.name].value;
       }
+      this.updateSearchOptions('');
     });
   }
 
   add(event: MatChipInputEvent): void {
     const input = event.input;
```

Comment:
This seems depricated and should be replaced with `const input = event.chipInput.inputElement;`